apps/condo/domains/contact/components/CreateContactForm.tsx

```diff
@@ -0,0 +1,246 @@
+import { Col, Form, Input, Row } from 'antd'
+import styled from '@emotion/styled'
+import { Rule } from 'rc-field-form/lib/interface'
+import React, { useEffect, useRef, useState } from 'react'
+import { useOrganization } from '@core/next/organization'
+import { useRouter } from 'next/router'
+import { useIntl } from '@core/next/intl'
+const { normalizePhone } = require('@condo/domains/common/utils/phone')
+import { Button } from '@condo/domains/common/components/Button'
+import { FormWithAction } from '@condo/domains/common/components/containers/FormList'
+import { ErrorsContainer } from '@condo/domains/contact/components/ErrorsContainer'
+import { PhoneInput } from '@condo/domains/common/components/PhoneInput'
+import { PropertyAddressSearchInput } from '@condo/domains/property/components/PropertyAddressSearchInput'
+import { UnitNameInput } from '@condo/domains/user/components/UnitNameInput'
+import { Contact } from '@condo/domains/contact/utils/clientSchema'
+
+const INPUT_LAYOUT_PROPS = {
+    labelCol: {
+        span: 6,
+    },
+    wrapperCol: {
+        span: 10,
+    },
+    style: {
+        paddingBottom: '12px',
+    },
+}
+
+const BottomLineWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+`
+
+export const CreateContactForm: React.FC = () => {
+    const intl = useIntl()
+    const FullNameLabel = intl.formatMessage({ id: 'field.FullName.short' })
```

Comment:
Missing translation for this key.
Please, add it into json-files

Reply:
I used this translation before in https://github.com/open-condo-software/condo/pull/229 . But separated PRs for easier review...and I guess some lang changes are there...they will work after both PRs will be merged 👍

Reply:
You can check this by rebasing this branch on top of 495th